packages/genesys-spark-components/src/components/beta/gux-inline-dropdown/example.html

```diff
@@ -0,0 +1,33 @@
+<h1>gux-inline-dropdown</h1>
+<form onchange="notify(event)" oninput="notify(event)">
+  <gux-inline-dropdown-beta>
+    <gux-listbox aria-label="status">
+      <gux-option-status-beta value="info" accent="info">
+        <span>Information</span>
```

Comment:
Just want to confirm that UX wants strings other than `Information`, `Success` etc... to be possible. If not we should handle the translations in the component.

Reply:
Yes I confirmed with UX that they want additional strings to be possible